multiples_sum.js

```diff
@@ -0,0 +1,23 @@
+function multSum(valOne, valTwo, max) {
+  var sumOne = 0;
+  var sumTwo = 0;
+  var numOne = 1;
+  var numTwo = 1;
```

Comment:
We want to avoid using `var` and convert these variable declarations to `let` to avoid variable scope problems. See [this article](https://sentry.io/answers/difference-between-let-and-var-in-javascript/#:~:text=The%20difference%20between%20let%20and,in%20which%20they're%20declared.) to better understand what I mean.